opentelemetry-user-events-logs/examples/README.md

```diff
@@ -0,0 +1,101 @@
+# Getting Started with OpenTelemetry Rust Logs in 5 Minutes - Exporting logs to user_events
+
+## Prerequisites
+
+* Rust and Cargo: [Install](https://doc.rust-lang.org/cargo/getting-started/installation.html) them if you haven't.
+* A Linux system with [user_events](https://docs.kernel.org/trace/user_events.html) enabled: Ubuntu 23.10 (once available), or kernel 6.4 or later build with user_events support.
+* Permissions: Make sure you're a Linux user with write permissions to the tracefs (preferably not root).
+* Perf Tool: Install the [perf](https://perf.wiki.kernel.org/index.php/Main_Page) tool to capture the user_events.
+* Decode-Perf Tool: Install the [decode-perf](https://perf.wiki.kernel.org/index.php/Main_Page) tool to decode the user_events.
```

Comment:
wrong link for decode-perf tool.

Reply:
Yeah, my bad :( Thanks for noticing this.